tests/zkasm/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "zkasm-test",
+  "version": "1.0.0",
+  "description": "zkASM Test Utils",
+  "scripts": {
+    "test": "node run-tests-zkasm.js"
+  },
+  "keywords": [
+  ],
+  "author": "Andrei Kashin",
+  "license": "AGPL",
+  "dependencies": {
+    "@0xpolygonhermez/zkasmcom": "https://github.com/0xPolygonHermez/zkasmcom.git#v1.0.0",
+    "yargs": "^17.5.1"
+  },
+  "devDependencies": {
+    "@0xpolygonhermez/zkevm-proverjs": "github:0xPolygonHermez/zkevm-proverjs#feature/64bits",
```

Comment:
It would seem to me like it is an ideal time to think about how our contribution story to the proverjs might look like in the future… :thinking:

Reply:
I agree that this will be important long-term to avoid duplicate maintenance efforts of the prover and processor code. I've added this as a topic for discussion on Monday: https://hackmd.io/MTmmOMOxShaMqMi83bxzRQ?view#November-20-2023

Right now, it doesn't make much sense to invest in de-duplication given an upcoming PIL2 upgrade that will change both the processor and prover code. What we could do is to advocate for PIL2 and related infra to allow for modularity/templating necessary for the code reuse.